net-p2p/transmission/transmission-2.93.recipe

```diff
@@ -4,13 +4,13 @@ encryption, a web interface, peer exchange, magnet links, DHT, µTP, UPnP and \
 NAT-PMP port forwarding, webseed support, watch directories, tracker editing, \
 global and per-torrent speed limits, and more."
 HOMEPAGE="https://www.transmissionbt.com/"
-COPYRIGHT="2005-2016 By the respective authors"
+COPYRIGHT="2005-2018 By the respective authors"
```

Comment:
The folks at Debian has done a great job at listing all [copyright holders][0], so you might want to use that.

[0]: http://metadata.ftp-master.debian.org/changelogs/main/t/transmission/transmission_2.92-3_copyright